react/week2/todoapp/src/App.js

```diff
@@ -0,0 +1,56 @@
+import React, { useState } from "react";
+import "./index.css";
+import ToDoList from './ToDoList'
+import Timer from './Timer.js'
+
+function App() {
+  const[inputlist,setInputList]=useState();
+  const[items,setItems]= useState([]);
+
+  const itemEvent=(event)=>{
+    setInputList(event.target.value);
+
+  }
+
+  const lisOfItems=()=>{
+    setItems((oldItem)=>{
+      return[...oldItem,inputlist];
+      
+    });
+    setInputList('');
+  }
+
+  const deleteItems=(id)=>{
+  
+    
+    setItems((oldItem)=>{
+      return oldItem.filter((arrElement, index)=>{
+              return index!==id;
+            });
+          });
```

Comment:
Regarding the naming conventions refer back to my previous comments but on the other hand it's a neat way of using the filter method.